packages/kolibri-common/composables/useBaseSearch.js

```diff
@@ -239,7 +238,7 @@ export default function useBaseSearch(descendant, store, router) {
   }
 
   function search() {
-    const currentBaseUrl = get(baseurl);
+    const currentBaseUrl = baseurl && get(baseurl);
```

Comment:
Is this check needed? My understanding of `get` is that it if given a non-reactive/ref value it will just return the value itself.

Reply:
Oh, didnt know it. Im gonna try this!